docs/index.rst

```diff
@@ -8,7 +8,7 @@ Welcome to the documentation for the Axelrod Python library
 
 Here is quick overview of the current capabilities of the library:
 
-* Over 100 strategies from the literature and some exciting original
+* Over 230 strategies including from the literature and some exciting original
```

Comment:
including many?

I had a thought that we could add a classifier, something like "from a publication" (not necessary for this PR, of course).

Reply:
> including many?

Thank you! Much better.

> 
> I had a thought that we could add a classifier, something like "from a publication" (not necessary for this PR, of course).

Not a bad idea and relatively straightforward with @gaffney2010 recent classifier work :+1: